README.md

```diff
@@ -1,2 +1,106 @@
-# getting-started
-A guide to hep new students set up their development environment.
+# Getting Started
+
+This is a guide to setting up a development environment for the Minnesota Pollinators projects, aimed at new students. You may already have some of the necessary programs installed&mdash;if so, you can just skip those steps!
+
+This document draws from prior writeups by [Paul Friederichsen][] and [Prof. Nic McPhee][].
+
+**Table of Contents**
+
+- [Creating a GitHub Account](#creating-a-github-account)
+- [Installing Visual Studio Code](#installing-visual-studio-code)
+- [Installing `git`](#installing-git)
+  - [On Windows](#on-windows)
+  - [On a Mac](#on-a-mac)
+- [Installing GitKraken](#installing-gitkraken)
+- [Installing Node.js](#installing-nodejs)
+
+## Creating a GitHub Account
+
+All of the code for Minnesota Pollinators&mdash;along with its revision history&mdash;is stored on [GitHub][], a website for hosting software projects. To work on Minnesota Pollinators, you'll need to sign up for a GitHub account [here][Sign up for GitHub].
+
+![GitHub's "sign up" page][i:Signing up for GitHub]
+
+You'll want to join the [Minnesota Pollinators][] organization ([Prof. KK Lamberty][] can invite you).
+
+If you're a student at Morris, be sure to sign up for the [student developer pack][Student developer pack] as well!
+
+## Installing Visual Studio Code
+
+Next, in order to read and write the code, you're going to need a text editor. There are a lot of good options, but the one that we've been using is [Visual Studio Code][], particularly because of its [live share][Live share] feature.
```

Comment:
```suggestion
Next, in order to read and write the code, you're going to need a text editor. There are a lot of good options, but the one that we've been using is [Visual Studio Code][], particularly because of its [Live Share][Live share] feature.
```

Reply:
Oh, is it a proper noun? Cool! :relaxed: